packages/autorest.python/tasks.py

```diff
@@ -44,6 +44,7 @@ class Config(NamedTuple):
 
 
 AUTOREST_DIR = os.path.dirname(__file__)
+AUTOREST_M4_VERSION = "--version=3.9.7 --use=@autorest/modelerfour@4.26.2 "
```

Comment:
the issue with this is I believe we'll easily not update when new versions are released. It's not as big of a deal since autorest is more stagnant now, but I don't think we should do this. If we can do `--version=latest` and `--use=@autorest/modelerfour@latest` I'm good, but otherwise I would just recommend updating your local autorest and m4 to latest if you find it's out of sync

Reply:
updated with `latest`